atomistics/workflows/evcurve/workflow.py

```diff
@@ -212,3 +205,13 @@ def get_thermal_properties(
             constant_volume=constant_volume,
             output_keys=output_keys,
         )
+
+    def _get_strains(self):
+        strains = self.strains
+        if strains is None:
+            strains = np.linspace(
+                -self.vol_range,
+                self.vol_range,
+                int(self.num_points),
+            )
+        return strains
```

Comment:
IMO this would read more naturally using the pythonic property pattern. I.e. after introducing `self._strains = None` in `__init__`

```suggestion
    @property
    def strains(self):
        if self._strains is None:
            self._strains = np.linspace(
                -self.vol_range,
                self.vol_range,
                int(self.num_points),
            )
        return self._strains
```

Reply:
I guess this is the same debate we had in other issues - `strains` to me is an internal function used by other classes derived from the `EnergyVolumeCurveWorkflow` still it is not a function that should be used by the user. So I would prefer to keep it a private function. It could be a private property named `_strains` but that would be an overlap with the already defined `_strains` variable, that is why I prefer `_get_strains()`.

Reply:
In this case "used by the user" would just be looking at what strains are applied, no? I find it strange that it's important to keep such data secret from the user.

Anyhow, we at least agree that the code as-is is a step in the right direction, so I have no objection to merging without this further (IMO) improvement being added.